### PR TITLE
add dimap custom chapter initial

### PR DESCRIPTION
**Pull request by chrieke**
Pull Request
============

Add custom Dimap chapter

### CODE REVIEW

File: source/up42-blocks/data/dimap-custom.rst [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+Block type: ``DATA``
+
+This block enables using DIMAP Pleiades and Spot data, stored in a bucket on *Google Cloud Storage (GCS)*
+or *Amazon Web Services (AWS)*, in a workflow on UP42.
+
+Within the bucket, the user can select specific images (via the scene ids) or search by
+location and time. The search can also be limited to a subfolder in the bucket via the
+`prefix` parameter.
+
```
Comment (rodrigoalmeida94) [inline source/up42-blocks/data/dimap-custom.rst:19]: One question I would have is where if the information for the data.json fetched from? Can I add my own data.json?

File: source/up42-blocks/data/dimap-custom.rst [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+* ``bucket_name``: The bucket name.
+* ``prefix``: A file structure prefix to limit the dataset search to a specific subdirectory. Conforms to the gcs & aws prefix structure,
+  which excludes the bucket name. E.g. `folder1/folder2/`.
+* ``ids``: An array of DIMAP Pleiades or Spot image identifiers. The ``ids`` filter overrides all other filters, e.g., ``intersects``, ``limit`` and/or ``time``.
+* ``time``: A date range to filter scenes on. This range applies to the acquisition date/time of the scenes.
+* ``bbox``: The bounding box to use as an AOI. Will return all scenes that intersect with this box. Use only ``box``
+  **or** ``intersects``.
+* ``intersects``: A GeoJSON geometry to use as an AOI. Will return all scenes that intersect with this geometry. Use
+  only ``intersects`` **or** ``bbox``.
+* ``contains``: A GeoJSON geometry to use as an AOI. Will return all scenes that completely cover this geometry. Use only ``contains``
```
Comment (markusuwe) [inline source/up42-blocks/data/dimap-custom.rst:45]: Geometry operations not fully consistent. Does the block actually support contains?

File: source/up42-blocks/data/dimap-custom.rst [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+            "limit": 4,
+            "max_cloud_cover": 20
+        }
+    }
+
+
+Output format
+-------------
+
+The output GeoJSON contains the Dimap file metadata, with the ``up42.data_path`` pointing to the
```
Comment (markusuwe) [inline source/up42-blocks/data/dimap-custom.rst:92]: Dangling sentence.

File: source/up42-blocks/data/dimap-custom.rst (hunk 1 above)
Comment (chrieke) [reply source/up42-blocks/data/dimap-custom.rst:19]: good point, added a sentence addressing that.

File: source/up42-blocks/data/dimap-custom.rst (hunk 3 above)
Comment (chrieke) [reply source/up42-blocks/data/dimap-custom.rst:92]: fixed, ty

File: source/up42-blocks/data/dimap-custom.rst (hunk 2 above)
Comment (chrieke) [reply source/up42-blocks/data/dimap-custom.rst:45]: Also needed to check, but yes we handle the contains case [in the code.](https://github.com/up42/blocks/blob/f6528ac8cda1ddcfb8e8ce67efd39989904c4f68/blocks/dimap-custom/src/dimap_custom.py#L238)

**Pull request merged**